src/graph/CMakeLists.txt

```diff
@@ -47,5 +46,9 @@ add_dependencies(
     stats_obj
 )
 
+add_library(
+    graph_http_obj OBJECT
+    GraphHttpHandler.cpp
+)
```

Comment:
Is there no dependency here?

Reply:
Just depend on base_obj.  Good catch

Reply:
Here is adding a library, OBJECT library, no dependencies.